app/controllers/pages/exit_page_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Pages::ExitPageController < PagesController
+  before_action :can_add_page_routing, only: %i[new create]
+  before_action :ensure_exit_pages_enabled
+
+  def new
+    exit_page_input = Pages::ExitPageInput.new(form: current_form, page:, answer_value: params[:answer_value])
```

Comment:
Is there way to make it clearer that answer_value should be passed as a param when redirecting users here?

I can't think of much beyond a comment and maybe a test in the controller suite to show what happens when it's not included?

Reply:
Thanks to @lfdebrux we've now got a guard clause - if you try to hit the `new` or `create` action in the exit page controller without an answer value, you'll get bounced to the new route page instead - so you can pick an answer value.